support/reftest/reftest

```diff
@@ -0,0 +1,249 @@
+#!/usr/bin/env python
+"""
+
+Prepare reference test data for Exodus-lambda integration test.
+It requires a "data.yml" file specifying the reference test data.
+
+"""
+
+import os
+import sys
+import yaml
+import boto3
+import hashlib
+import tempfile
+import requests
+import argparse
+
+from tqdm import tqdm
+from datetime import datetime
+
+
+class DBHandler:
+    def __init__(self, table, session):
+        self.dynamodb = session.client("dynamodb")
+        self.table = table
+
+    def put_item(self, web_uri, object_key, from_date=None, metadata={}):
+        if not from_date:
+            from_date = datetime.utcnow().isoformat(timespec="seconds")
```

Comment:
Minor: rather than calculating from_date internally here, it'd probably be better to calculate it only once for the runtime of the process.  That way you can easily find the entire set of entries added by a single run of the process as they'll all have the same timestamp.